commandapi-core/src/main/java/dev/jorel/commandapi/CommandAPICommand.java

```diff
@@ -85,6 +85,16 @@ public CommandAPICommand withSubcommand(CommandAPICommand subcommand) {
 		this.subcommands.add(subcommand);
 		return this;
 	}
+	
+	/**
+	 * Adds a subcommand to this command builder
+	 * @param subcommand the subcommand to add as a child of this command 
```

Comment:
This JavaDocs comment needs updating to account for plurals (e.g. "Adds subcommands to ...")